config/ftbquests/quests/chapters/occultism.snbt

```diff
@@ -1705,17 +1729,15 @@
 				"",
 				"The Drikwing is an otherworldly bird that imparts special effects on its master when close.",
 				"",
-				"Tip: There is a ritual you can use to summon a wild parrot that can make the sacrifice needed for this ritual much easier.",
-				"",
-				"Tip #2: You must summon the Drikwing Familiar, the Wild Drikwing Summon will not work for this quest."

```

Comment:
It seems you accidentally removed some content here. It was added in-dev in preparation for 0.4.11, so it wouldn't have been on the master branch, that may be the reason :)